src/commandsAndMenu.tsx

```diff
@@ -60,6 +60,10 @@ export interface IGitCloneArgs {
    * Git repository url
    */
   url: string;
+  /**
+   * Git metadata checked
+   */
+  metaDataChecked: boolean;
```

Comment:
Let's rename this one `versioning` too

```suggestion
   * Whether to activate git versioning in the clone or not.
   * If false, this will remove the .git folder after cloning.
   */
  versioning: boolean;
```